### PR TITLE
Updates to Developer Guide

### PR DESCRIPTION
**Pull request by kianhong95**

### CODE REVIEW

File: src/seedu/addressbook/commands/employee/EmployeeDeleteCommand.java [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
+ * Deletes an employee identified using it's last displayed index from the Rms.
  */
 public class EmployeeDeleteCommand extends Command{
 
     public static final String COMMAND_WORD = "delemp";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
             + "Deletes the employee identified by the index number used in the last employee listing.\n\t"
-            + "Parameters: INDEX\n\t"
+            + "Parameters: INDEX (must be a positive integer)\n\t"
```
Comment (px1099) [inline src/seedu/addressbook/commands/employee/EmployeeDeleteCommand.java]: I don't think the note in the bracket is useful. Instead, you can write it in the user guide

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -828,11 +828,13 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
 |`* *` |Manager |add, edit and delete items from the menu |keep my restaurant menu interesting
 
-|`* *` |Manager |view employees  |know which one of my employees are working today
+|`* *` |Manager |list employees  |to view the details of the employees working in the restaurant
 
-|`* * *` |Chef |view all the orders made  |know which order I need to prepare next
+|`* * *` | Manager | add, edit and delete employees from the system |to manage information of the employees working in the restaurant
 
-|`* * *` |Chef |update the order status  |let the customers know that their orders are ready
+|`* * *` |view all the orders made |Chef  |know which order I need to prepare next
```
Comment (SalsabilTasnia) [inline docs/DeveloperGuide.adoc]: Please swap the second and third column here.

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -301,7 +343,56 @@ private Command prepareEmployeeDelete(String args) {
...
+    }
+
+    /**
+     * Returns new information to be edited if it is not empty
+     */
+    private static String checkEmpty(String toCheck, String argument){
+        if (toCheck == null || toCheck.isEmpty()) {
+            switch(argument) {
+                case "phone":
+                    toCheck = "00000000";
```
Comment (px1099) [inline src/seedu/addressbook/parser/Parser.java]: This looks outdated compared to upstream/master
You might need to merge upstream/master into your branch to update the latest changes

**Review by px1099 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (kianhong95) [reply docs/DeveloperGuide.adoc]: Done.

File: src/seedu/addressbook/parser/Parser.java (hunk 3 above)
Comment (kianhong95) [reply src/seedu/addressbook/parser/Parser.java]: Done.

**Review by px1099 — APPROVED** [review]

**Pull request merged**